.travis.yml

```diff
@@ -2,8 +2,11 @@ env:
   - CC=clang CXX=clang++ npm_config_clang=1
 language: node_js
 before_install:
-- npm install -g npm@2
+- npm install -g npm@5
 node_js:
-- '0.10'
 - '4'
 - '5'
+- '6'
+- '7'
+- '8'
+- 'node'
```

Comment:
I think this can be just `- 4` to test on node 4, to not run too many travis tasks

Reply:
@iefserge how about `4` and `node` so we at least make sure it's working with latest release?